config.w32

```diff
@@ -103,9 +103,11 @@ if (PHP_MONGODB != "no") {
     /I" + configure_module_dirname + "/src/MongoDB/Monitoring \
     /I" + configure_module_dirname + "/src/contrib \
     /I" + configure_module_dirname + "/src/libmongoc/src/common \
+    /I" + configure_module_dirname + "/src/libmongoc/src/uthash \
     /I" + configure_module_dirname + "/src/libmongoc/src/libbson/src \
     /I" + configure_module_dirname + "/src/libmongoc/src/libbson/src/jsonsl \
     /I" + configure_module_dirname + "/src/libmongoc/src/libmongoc/src \
+    /I" + configure_module_dirname + "/src/libmongoc/src/libmongoc/src/mongoc \
```

Comment:
This was the suggested workaround for libmongoc using `<>` instead of `""` for some new `#include` statements introduced in 1.27.0. CI suggests this is only necessary for Windows, but let me know if you think it'd be wise to duplicate this in `config.m4`. Alternatively, we can wait and see if any bug reports come in and address this in a patch release if needed (any failure would be quite obvious).

Reply:
I think it's fine to do this for Windows only at this time 👍